db/migrate/20180111145635_add_fields_to_user.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToUser < ActiveRecord::Migration[5.1]
```

Comment:
Maybe you should consider defining a profile model where you can store details about a user, and keep only essential data in the user table, such as username/email and password.

Reply:
I updated this, it's now in profile model